pkg/datastore/mysql/ensurer/indexes.sql

```diff
@@ -96,19 +96,23 @@ CREATE INDEX event_project_id_updated_at_desc ON Event (ProjectId, UpdatedAt DES
 
 -- index on `EventKey` ASC, `Name` ASC, `ProjectId` ASC and `CreatedAt` DESC
 ALTER TABLE Event ADD COLUMN EventKey VARCHAR(64) GENERATED ALWAYS AS (data->>"$.event_key") VIRTUAL NOT NULL, ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL NOT NULL;
-CREATE INDEX event_key_name_project_id_created_at_desc ON Event (EventKey, Name, ProjectId, CreatedAt DESC);
+CREATE INDEX event_event_key_name_project_id_created_at_desc ON Event (EventKey, Name, ProjectId, CreatedAt DESC);
+
+-- index on `ProjectId` ASC, `Status` ASC, CreatedAt DESC
+ALTER TABLE Event ADD COLUMN Status INT GENERATED ALWAYS AS (data->>"$.status") VIRTUAL NOT NULL;
+CREATE INDEX event_project_id_status_created_at_desc ON Event (ProjectId, Status, CreatedAt DESC);
 
 -- index on `ProjectId` ASC, `Status` ASC, UpdatedAt DESC
 ALTER TABLE Event ADD COLUMN Status INT GENERATED ALWAYS AS (data->>"$.status") VIRTUAL NOT NULL;
-CREATE INDEX project_id_status_updated_at_desc ON Event (ProjectId, Status, CreatedAt DESC);
+CREATE INDEX event_project_id_status_updated_at_desc ON Event (ProjectId, Status, UpdatedAt DESC);
 
 -- index on `Name` ASC, `ProjectId` ASC, `UpdatedAt` DESC
 ALTER TABLE Event ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL NOT NULL;
-CREATE INDEX event_key_name_project_id_created_at_desc ON Event (Name, ProjectId, UpdatedAt DESC);
+CREATE INDEX event_name_project_id_updated_at_desc ON Event (Name, ProjectId, UpdatedAt DESC);
 
 -- index on `Name` ASC, `ProjectId` ASC, `Status` ASC, UpdatedAt DESC
 ALTER TABLE Event ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL NOT NULL, ADD COLUMN Status INT GENERATED ALWAYS AS (data->>"$.status") VIRTUAL NOT NULL;
-CREATE INDEX name_project_id_status_updated_at_desc ON Event (Name, ProjectId, Status, UpdatedAt DESC);
+CREATE INDEX event_name_project_id_status_updated_at_desc ON Event (Name, ProjectId, Status, UpdatedAt DESC);
```

Comment:
This will create a new index for the same one.
Do we need to drop the old index?
Or should use a rename index command instead.

Reply:
Actually, it's not released yet, so I feel like that's fine.

Reply:
Got it. If all of them are not released yet, no problem. 👍